AnalysesFolder/isDecreasingTest/ProcessFlightData.cpp

```diff
@@ -0,0 +1,97 @@
+/*
+ * ProcessFlightData.c
+ *
+ *  Created on: Mar 4, 2024
+ *      Author: sethm
+ */
+
+// When uploading to the payload bay micro, change this to true and the other INO to false
+
+#include "bufferCopy.h"
+#include <iostream>
+#include <vector>
+#include <sstream>
+#include <fstream>
+#include <string>
+#include <algorithm>
+#include <ctime>
+#include <random>
+
+using namespace std;
+
+
+enum Mode {
+  BELOW_5K,
+  WATCHING,
+  PAST_APOGEE
+};
+
+string grabNextField(string& csvLine) {
+	size_t len;
+	string field = "";
+	while (csvLine[0] == '"') {
+		csvLine.erase(0, 2);
+		len = csvLine.find('"');
+		field += csvLine.substr(0, len);
+		csvLine.erase(0, len + 1);
+	}
+	len = csvLine.find(',');
+	field += csvLine.substr(0, len);
+	csvLine.erase(0, len + 1);
+	return field;
+}
+
+int main() {
+	srand(time(NULL));
+	std::default_random_engine rng(rand());
+	const float STDEV = 1.5f;
```

Comment:
I assume this value is still waiting on @TheRoyalIrishman's analysis?

Reply:
yup, just left it at 1.5, as 3 standard deviations out is considered to be all points, and our largest was 4.5 ft. Whenever Cam gives us numbers we'll update

Reply:
1.4432132 is approximately our standard deviation